kolibri/plugins/management/assets/src/views/manage-content-page/channels-grid.vue

```diff
@@ -65,6 +65,13 @@
     >
       {{ $tr('deleteFailureNotification') }}
     </ui-alert>
+    <ui-alert
```

Comment:
I wonder if snackbars would be a better alternative to these ui-alerts. Also, I don't know if the alert for `deleteSuccess` should have the same color as `importSuccess`. Thoughts @jtamiace @khangmach ?